docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,533 @@
+                            "score": "uncertainty_entropy" # change to your preferred score
+                        },
+                        "strategy": {
+                            "type": "WEIGHTS"
+                        }
+                    }
+                ]
+            }
+
+        .. note:: This works as well for Image Classifciation or Segmentation!
+
+    .. tab:: Visual Diversity and Active Learning (CORAL)
+
+        For combining two strategies, just specify both of them:
+
+        .. code-block:: python
+
+            {
+                "nSamples": 100, # set to the number of samples you want to select
+                "strategies": [
+                    {
+                        "input": {
+                            "type": "EMBEDDINGS"
+                        },
+                        "strategy": {
+                            "type": "DIVERSIFY"
+                        }
+                    },
+                    {
+                        "input": {
+                            "type": "SCORES",
+                            "task": "my_object_detection_task", # change to your task
+                            "score": "uncertainty_entropy" # change to your preferred score
+                        },
+                        "strategy": {
+                            "type": "WEIGHTS"
+                        }
+                    }
+                ]
+            }
+
+    .. tab:: Metadata Thresholding
+
+        This can be used e.g. to remove blurry images, which equals choosing 
+        samples whose sharpness is over a threshold:
+
+        .. code-block:: python
+
+            {
+                "nSamples": 100, # set to the number of samples you want to select
+                "strategies": [
+                    {
+                        "input": {
+                            "type": "METADATA", 
+                            "key": "lightly.sharpness"
+                        },
+                        "strategy": {
+                            "type": "THRESHOLD",
+                            "threshold": 20,
+                            "operation": "BIGGER"
+                        }
+                    }
+                ]
+            }
+
+    .. tab:: Score Thresholding
+
+        You can specify to only choose images that have at least 2 objects in them:
+
+        .. code-block:: python
+
+            {
+                "nSamples": 100, # set to the number of samples you want to select
+                "strategies": [
+                    {
+                        "input": {
+                            "type": "SCORES",
+                            "task": "my_object_detection_task", # change to your task
+                            "score": "object_frequency"
+                        },
+                        "strategy": {
+                            "type": "THRESHOLD",
+                            "threshold": 2,
```

Comment:
the score is normalized to `[0, 1]` so this doesn't work the way it's intended to

Reply:
Damn. I thought I removed this example